htdocs/core/lib/modulebuilder.lib.php

```diff
@@ -620,3 +620,48 @@ function writePropsInAsciiDoc($file, $objectname, $destfile)
 	}
 	return 1;
 }
+
+/**
+ * Search a string and return all lines needed from file
+ * @param  string  $file    file for searching
+ * @param  string  $start   start line if exist
+ * @param  string  $end     end line if exist
+ * @return string           return the content needed
+ */
+function getFromFile($file, $start, $end)
+{
+	$i = 1;
+	$keys = array();
+	$lines = file($file);
+	// Search for start and end lines
+	foreach ($lines as $i => $line) {
+		if (strpos($line, $start) !== false) {
+			// Copy lines until the end on array
+			while (($line = $lines[++$i]) !== false) {
+				if (strpos($line, $end) !== false) {
+					break;
+				}
+				$keys[] = $line;
+			}
+			break;
+		}
+	}
+	$content = implode("", $keys);
+	return $content;
+}
+
+/**
+ * Delete property from documentation if we delete object
+ * @param  string  $file         file or path
+ * @param  string  $objectname   name of object wants to deleted
+ * @return void
+ */
+function deletePropsFromDoc($file, $objectname)
+{
+
+	$start = "== Table of fields and their properties for object *".ucfirst($objectname)."* : ";
+	$end = "== end table for object ".ucfirst($objectname);
+	$string = getFromFile($file, $start, $end);
```

Comment:
Here use fileGetContents($file) to get the file in a string
Then do a preg_replace() with a regex that starts with $start and ends with $end and that takes everything in beetween the 2, and replace the match.
The call looks like : preg_replace($regex, '', $stringtomodify)

Reply:
ok, it's done